aopy/postproc/base.py

```diff
@@ -149,6 +149,55 @@ def get_trial_targets(trials, targets):
         trial_targets[trial].append(targets[idx])
     return trial_targets
 
+def get_minimum_trials_per_target(target_idx, cond_mask):
+    '''
+    Get the minimum number of trials per target after restricting trials
+    
+    Args:
+        target_index (ntr): target index
+        cond_mask (ntr): boolean array to remove trials
+        
+    Returns:
+        (int): minimum number of trials per target
+        
+    '''
+    
+    num_trial_targ = []
+    for itarget in np.unique(target_idx):
+        num_trial_targ.append(sum(target_idx[cond_mask] == itarget))
+    min_trial = min(num_trial_targ)
```

Comment:
you can speed this up with list comprehension. I think this should work:

`min_trial = min([sum(target_idx[cond_mask] == itarget) for itarget in np.unique(target_idx)])`